tests/phpunit/Helpers.php

```diff
@@ -56,3 +64,46 @@ function generator_to_phpunit_data_provider(iterable $source): Generator
         yield $key => [$value];
     }
 }
+
+/**
+ * Creates a temporary directory.
+ *
+ * @param string $namespace the directory path in the system's temporary directory
+ * @param string $className the name of the test class
+ *
+ * @return string The path to the created directory
+ *
+ * @TODO: extract the FS utils from Box if they are not going to be merged to Symfony
```

Comment:
I'll work on this TODO in the coming month, TLDR we'll have a richer file system class